### PR TITLE
Distro info fix

### PR DESCRIPTION
**Pull request by lucasmoura**

### CODE REVIEW

**Review by renanrodrigo — APPROVED** [review]
lgtm

File: debian/changelog [hunk 1]
```diff
@@ -2,9 +2,11 @@ ubuntu-advantage-tools (27.2~21.10.1) impish; urgency=medium
 
   * d/control:
     - add comments to explain complex build-depends
-    - add version requirement to distro-info (LP: #1932028)
+    - remove unnecessary distro-info dependency from build-depends
   * d/tools.postinst:
     - run status.json schema patch script to avoid non-root status errors
+  * d/rules:
+    - add version requirement to distro-info (LP: #1932028)
   * New upstream release 27.2:
```
Comment (blackboxsw) [inline debian/changelog:10]: You are now missing a new comment entry for proxy manpage updates for proxy setup

Comment (blackboxsw) [reply debian/changelog:10]: Disregard this, we already represent that content change in the changelog anyway
 + add proxy config options to man page

**Review by blackboxsw — APPROVED** [review]
+1

**Pull request merged**